packages/vue-loader/test/test.js

```diff
@@ -29,30 +23,27 @@ const testFixture = fixture => {
               }
             },
             {
-              loader: path.resolve(__dirname, '..'),
-              options: {
-                compilers: [VueJSXCompiler]
-              }
+              loader: path.resolve(__dirname, '..')
             }
           ]
         }
       ]
     }
   })
 
-  compiler.outputFileSystem = new MemoryFs()
+  compiler.outputFileSystem = createFsFromVolume(new Volume())
 
   return new Promise((resolve, reject) => {
     compiler.run((err, stats) => {
       if (err) reject(err)
-      const module = stats.toJson().modules.find(m => m.name === fileName)
-        .source
-      resolve(module)
+      if (stats.hasErrors()) reject(new Error(stats.toJson().errors))
```

Comment:
I ended up not running this because it was a little confusing to get webpack to obey me. I kept getting some `fs.join` errors that were beyond my current ability to quickly debug. so I skipped the test assertion below.

Well it wasn't running before so... 😅 Happy to get contributions toward this!